charts/policy-service/values.yaml

```diff
@@ -20,7 +20,7 @@ service:
 
 image:
   name: policy-service
-  tag: SNAPSHOT-026cffd
```

Comment:
More a question than anything else.  This tag is only being changed in this one service and yet this same tag appears in the other services with the older value.  I would expect that this will be the same number for all of them.  Can I assume that the tags for all the services get updated at some other point?

Reply:
The other services do not need to have their tag updated as there are no changes to them. The tags should only be updated on the service(s) where there are changes.

Reply:
Having said that there are no real changes to the service but the `pi` profile was used to make sure the `deployment.yaml` changes were used when deploying the service.